Core/src/main/java/org/tribuo/onnx/ONNXOperators.java

```diff
@@ -75,6 +78,26 @@
      * Element-wise exponentiation with broadcasting.
      */
     POW("Pow",2,1),
+    /**
+     * Hardmax(element in input, axis) = 1 if the element is the first maximum value along the specified axis, 0 otherwise.
```

Comment:
Unless 'first' here refers to something specific.
```suggestion
     * Hardmax(element in input, axis) = 1 if the element is the maximum value along the specified axis, 0 otherwise.
```

Reply:
Actually that's from the ONNX spec for the operator, it only sets the value to 1 for the first maximum element - https://github.com/onnx/onnx/blob/master/docs/Operators.md#Hardmax.